src/common/hooks/use-wallet.ts

```diff
@@ -64,10 +68,10 @@ export const useWallet = () => {
         const newNonce = tx.auth.spendingCondition?.nonce.toNumber();
         if (newNonce !== undefined) {
           set(apiRevalidation, current => (current as number) + 1);
-          const blockHeight = await snapshot.getPromise(latestBlockHeightStore);
-          const network = await snapshot.getPromise(currentNetworkStore);
-          const address = await snapshot.getPromise(currentAccountStxAddressStore);
-          set(latestNoncesStore([network.url, address || '']), () => ({
+          const blockHeight = await snapshot.getPromise(latestBlockHeightState);
```

Comment:
I know this was here before, but what is the diff b/w `latest` and `current`? If nothing, maybe all should be `current`?

Reply:
just semantics between different people, will update